### ISSUE TITLE
WiFiDirectServices Advertisement keeps aborting

### ISSUE THREAD

**Issue by MrBlaTi** [Microsoft/Windows-universal-samples#429]

**Comment by wernerdenzin** [comment]
Yes! I have the same problem! This is a critical issue, because we can't move on! Please, help us!

**Comment by Yorchito** [comment]
@MrBlaTi @wernerdenzin  Could you please run the following command on PowerShell:

netsh wlan show wirelesscapabilities

"Action Frame" and "P2P Service Name Discovery" need to be both supported
